js/components/root.js

```diff
@@ -25,13 +25,13 @@
 		'	<template v-if="missing.length">' +
 		'		<h3>' + t('caniupdate', 'Missing updates') + '</h3>' +
 		'		<ul>' +
-		'			<li v-for="app in missing">{{app.appName}} <a :href="\'https://apps.nextcloud.com/apps/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
+		'			<li v-for="app in missing">{{app.appName}} <a :href="\'https://apps.nextcloud.com/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
 		'		</ul>'+
 		'	</template>' +
 		'	<template v-if="available.length">' +
-		'		<h3>' + t('caniupdate', 'Available updates') + '</h3>' +
+		'		<h3>' + t('caniupdate', 'Available new versions') + '</h3>' +
 		'		<ul>' +
-		'			<li v-for="app in available">{{app.appName}} <a :href="\'https://apps.nextcloud.com/apps/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
+		'			<li v-for="app is available">{{app.appName}} <a :href="\'https://apps.nextcloud.com/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
```

Comment:
the `is` is JS magic syntax hence you break the listing of the updates....

Reply:
you are breaking the URL by removing the `apps/` part

Reply:
Could you explain "app in available" or what happens to it?

Reply:
https://vuejs.org/v2/guide/list.html#v-for-with-an-Object